<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogTask</class>
 <widget class="QDialog" name="DialogTask">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Elapsed Time</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Activity</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QTimeEdit" name="timeEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButtonInsert">
       <property name="text">
        <string>Insert</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="QTableView" name="tableView"/>
     </item>
     <item row="2" column="1">
      <widget class="QTextEdit" name="textEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Note</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogTask</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>318</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogTask</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>318</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonInsert</sender>
   <signal>clicked()</signal>
   <receiver>DialogTask</receiver>
   <slot>insert()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>386</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>12</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableView</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>DialogTask</receiver>
   <slot>selectedRow(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>376</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>429</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textEdit</sender>
   <signal>textChanged()</signal>
   <receiver>DialogTask</receiver>
   <slot>noteChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>insert()</slot>
  <slot>selectedRow(QModelIndex)</slot>
  <slot>noteChanged()</slot>
 </slots>
</ui>
